contracts/contracts/token/BridgedWOETH.sol

```diff
@@ -54,6 +54,14 @@ contract BridgedWOETH is
         _burn(account, amount);
     }
 
+    /**
+     * @dev Burns tokens from `msg.sender`
+     * @param amount Amount of tokens to burn
+     */
+    function burn(uint256 amount) external onlyRole(BURNER_ROLE) nonReentrant {
```

Comment:
just a though, not something that we should address in this PR: 

should we avoid the same method names with different parameter values? The way I see it, this way is cleaner API wise, but harder for web3 users. Our devs surely won't have a problem with it, but sometimes it is unintuitive that via ethers in js one must call the function via `contractName['function signature']` instead of a `contractName.functionName`.  In this case smth like `burn` & `burnFrom` would probably work

Curious what your thoughts are on this @shahthepro, @DanielVF , @naddison36?

Reply:
Good point. Let me check what methods different versions of CCIP Token Pool call, I can change it later